src/main/java/org/fogbowcloud/manager/core/intercomponent/xmpp/handlers/RemoteDeleteOrderRequestHandler.java

```diff
@@ -23,6 +23,8 @@ public RemoteDeleteOrderRequestHandler() {
 
     @Override
     public IQ handle(IQ iq) {
+        LOGGER.info("Received request for order: " + iq.getID());
```

Comment:
not sure if this log line should be info or debug. it is important to be consistent along the code